src/scss/components/_price-variation.scss

```diff
@@ -7,13 +7,16 @@
 **/
 
 .price-variation {
-  @include flexbox;
-  @include flex-direction(row);
-  @include flex-wrap(wrap);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  position: relative;
```

Comment:
Ok. Let's check on another browsers if there is no side effect (triborg have this component)